lib/katello/engine.rb

```diff
@@ -226,6 +226,10 @@ class Engine < ::Rails::Engine
       ::AuditSearch::ClassMethods.prepend Katello::Concerns::AuditSearch
       ::Widget.singleton_class.prepend Katello::Concerns::WidgetExtensions::ClassMethods
 
+      if with_remote_execution?
```

Comment:
This is throwing an error - it should be Katello.with_remote_execution

Reply:
Right, not sure how I managed to let this slip